web/apps/labelstudio/src/pages/CreateProject/Import/Import.jsx

```diff
@@ -374,42 +402,66 @@ export const ImportPage = ({
-                  <dt>Audio</dt>
-                  <dd>{supportedExtensions.audio.join(", ")}</dd>
-                  <dt>Video</dt>
-                  <dd>mpeg4/H.264 webp, webm* {/* Keep in sync with supportedExtensions.video */}</dd>
-                  <dt>Images</dt>
-                  <dd>{supportedExtensions.image.join(", ")}</dd>
-                  <dt>HTML</dt>
-                  <dd>{supportedExtensions.html.join(", ")}</dd>
-                  <dt>Time Series</dt>
-                  <dd>{supportedExtensions.timeSeries.join(", ")}</dd>
-                  <dt>Common Formats</dt>
-                  <dd>{supportedExtensions.common.join(", ")}</dd>
-                </dl>
-                <b>
-                  * – Support depends on the browser
-                  <br />* – Direct media uploads have{" "}
-                  <a href="https://labelstud.io/guide/tasks.html#Import-data-from-the-Label-Studio-UI">limitations</a>{" "}
-                  and we strongly recommend using{" "}
-                  <a href="https://labelstud.io/guide/storage.html" target="_blank" rel="noreferrer">
-                    Cloud Storage
+            <div className="flex gap-4 justify-center items-start">
+              <label htmlFor="file-input">
+                <div className={dropzoneClass.elem("content")}>
+                  <header>
+                    Drag & drop files here
+                    <br />
+                    or click to browse
+                  </header>
+                  <IconUpload height="64" className={dropzoneClass.elem("icon")} />
+                  <dl>
+                    <dt>Text</dt>
+                    <dd>{supportedExtensions.text.join(", ")}</dd>
+                    <dt>Audio</dt>
+                    <dd>{supportedExtensions.audio.join(", ")}</dd>
+                    <dt>Video</dt>
+                    <dd>mpeg4/H.264 webp, webm* {/* Keep in sync with supportedExtensions.video */}</dd>
+                    <dt>Images</dt>
+                    <dd>{supportedExtensions.image.join(", ")}</dd>
+                    <dt>HTML</dt>
+                    <dd>{supportedExtensions.html.join(", ")}</dd>
+                    <dt>Time Series</dt>
+                    <dd>{supportedExtensions.timeSeries.join(", ")}</dd>
+                    <dt>Common Formats</dt>
+                    <dd>{supportedExtensions.common.join(", ")}</dd>
+                  </dl>
+                  <b>
+                    * – Support depends on the browser
+                    <br />* – Direct media uploads have{" "}
+                    <a href="https://labelstud.io/guide/tasks.html#Import-data-from-the-Label-Studio-UI">limitations</a>{" "}
+                    and we strongly recommend using{" "}
+                    <a href="https://labelstud.io/guide/storage.html" target="_blank" rel="noreferrer">
+                      Cloud Storage
+                    </a>{" "}
+                    instead
+                  </b>
```

Comment:
we need to hide these documentation links for white labeled environments

Reply:
it's OSS and the code is not shared with LSE

Reply:
ahh ok, we have another import component in LSE 👍